hub/powertoys/run.md

```diff
@@ -104,23 +104,28 @@ The PowerToys Run settings menu includes a plugin manager that allows you to ena
 
 PowerToys Run enables a set of system level actions that can be executed.
 
-| Action command / Search result | Action | Note |
-| :--- | :--- | :--- |
-| `Shutdown` | Shuts down the computer | |
-| `Restart` | Restarts the computer | |
-| `Sign Out` | Signs current user out | |
-| `Lock` | Locks the computer | |
-| `Sleep` | Sleeps the computer | |
-| `Hibernate` | Hibernates the computer | |
-| `Empty Recycle Bin` | Empties the recycle bin | |
-| `UEFI Firmware Settings` | Reboot computer into UEFI Firmware Settings | Only available on systems with UEFI firmware.<br />(Requires administrative permissions.) |
-| `IP address` | Shows the ip addresses from the network connections of your computer. | The search query has to start with `IP` or `address`. |
-| `MAC address` | Shows the mac addresses from the network adapters in your computer. | The search query has to start with `MAC` or `address`. |
+> [!TIP]
+> By default you can search the system commands by their localized name if your system language is supported by PowerToys application.
+> But if you prefere to search them by their english names you can enabled the non-localized search by changing the plugin setting **'Use localized system commands instead of English ones'** to **disabled** in the [plugin manager](#plugin-manager).
 
+| Command / Search result | Localized command name | Action | Note |
+| :--- | :--- | :--- | :--- |
+| `Shutdown` | _Shutdown_ | Shuts down the computer | |
+| `Restart` | _Restart_ | Restarts the computer | |
+| `Sign Out` | _Sign Out_ | Signs current user out | |
+| `Lock` | _Lock_ | Locks the computer | |
+| `Sleep` | _Sleep_ | Sleeps the computer | |
```

Comment:
Puts the computer to sleep

Reply:
Typo exist since ever and no one found it till now. 😅